README.md

```diff
@@ -92,13 +110,11 @@ Are you interested in contributing to kops? We the maintainers and community wou
 
 ## Office Hours
 
-Kops maintainers also have time (1 hour) set aside every other week to offer help and guidance to the
-community. This is the maintainer's **office hours**. During this time we might work with newcomers, help with PRs, and discuss new features. Anything goes.
+Kops maintainers set aside 1 hour every other week for **public** office hours for a video chat in which we strive to get to know the other developers either working on the project or interested in getting to know more about it. We tend to use this hour to discuss the current state of the kops project, to strategize about how to move it forward, ti discuss open and upcoming PRs, to demo, and to offer help and guidance to the community. Generally this time is focused on developers, although we will never turn a courteous participant away. Even if you've never actually installed kops, we're interested to have you stop by our office hours.
```

Comment:
How about (we could also link the when section below, but I do not want to bury it and to repeat it may be ok):

Begin edit
-----

Kops maintainers set aside one hour every other week for **public** office hours. Office hours are hosted on a [zoom video chat](https://zoom.us/my/k8ssigaws) on Fridays at [5 pm UTC/12 noon ET/9 am US Pacific](http://www.worldtimebuddy.com/?pl=1&lid=100,5,8,12), on odd week numbered weeks.  We strive to get to know and help developers either working on `kops` or interested in getting to know more about the project.

### Open Office Hours Topics

Include but not limited to:

- Help and guide to those who attend, who are interested in contributing.
- Discuss the current state of the kops project, including releases.
- Strategize about how to move `kops` forward.
- Collaborate about open and upcoming PRs.
- Present demos.

This time is focused on developers, although we will never turn a courteous participant away. Please swing by, even if you've never actually installed kops.